mirgecom/diffusion.py

```diff
@@ -204,6 +204,10 @@ def get_diffusion_flux(self, discr, quad_tag, dd, alpha, grad_u):  # noqa: D102
         return discr.project(dd_quad, dd_allfaces_quad, flux_quad)
 
 
+class _DiffusionStateTag:
+    pass
```

Comment:
Shouldn't diffusion use one unique tag per message?  Currently it uses this tag for all its messages.